package.json

```diff
@@ -48,6 +48,7 @@
     "quasar": "^1.9.1",
     "sass-loader": "^7.1.0",
     "shepherd.js": "^2.10.0",
+    "vue-clipboard2": "^0.3.1",
```

Comment:
Can you please check if the method in `CopyToClipboard.vue:46` would work? in the end they use the same method. We could save the extra dependencies, see `package-lock.json`.